fed/_private/compatible_utils.py

```diff
@@ -20,6 +21,23 @@
 from fed._private import constants
 
 
+def _get_package_version_string(package_name):
```

Comment:
```suggestion
def get_package_version(package_name: str) -> str:
```

Reply:
Move this function to [utils.py](https://github.com/ray-project/rayfed/blob/main/fed/utils.py)